development/build_images.sh

```diff
@@ -1,13 +1,9 @@
 #!/usr/bin/env bash
+# shellcheck source=../ci/lib/stdlib.bash
+source "$(dirname "${BASH_SOURCE[0]}")/../ci/lib/stdlib.bash" || exit 67
 
-set -xeu
-
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-INTEGRATION_TEST_DIR="$SCRIPT_DIR/../integration-tests/"
```

Comment:
I'm guessing this is no longer needed because the helper? stdlib.bash? (haven't fully combed through the code yet)

Reply:
Yes, that's right. I can mark up the stdlib.bash with some line comments to talk through what's going on, if you like?